basic-javascript/project-1.spec.js

```diff
@@ -1,3 +1,300 @@
-const helpers = require('./project-1');
+const {
+    multiplyByTen,
+    subtractFive,
+    areSameLength,
+    areEqual,
+    lessThanNinety,
+    greaterThanFifty,
+    add,
+    subtract,
+    divide,
+    multiply,
+    getRemainder,
+    isEven,
+    isOdd,
+    square,
+    cube,
+    raiseToPower,
+    roundNumber,
+    roundUp,
+    addExclamationPoint,
+    combineNames,
+    getGreeting,
+    getRectangleArea,
+    getTriangleArea,
+    getCircleArea,
+    getRectangularPrismVolume
+    } = require('./project-1');
 
 // start testing!
+
+describe('Jest',() => {
+    test('Jest is Running',()=>{
+
+    })
+});
+
+describe('multiply by ten',() => {
+    test('multiplys correctly',()=>{
+        const input = 5;
+        const expected = 50;
+        const result = multiplyByTen(input);
+        expect(result === expected)
+    })
+})
+
+describe('Subtract Five',() => {
+    test('minus five', () => {
+        const input = 5;
+        const expected = 0;
+        const result = subtractFive(input);
+        expect(expected === result)
+    })
+})
+
+describe('Strings are the Same Length', () => {
+    test('same length',() => {
+        const i1 = i2 = "same length?";
+        const result = areSameLength(i1,i2);
+        expect(result === true)
+    })
+})
+
+describe('are Equal',()=>{
+    test('Strict equals',() => {
+        const i1 = i2 = 2;
+        const expected = true;
+        const result = areEqual(i1,i2);
+        expect(result === expected);
```

Comment:
for functions that return booleans, you want to test both the true and the false case